tests/test_toxenv.py

```diff
@@ -116,6 +116,13 @@
 class TestToxEnv:
     """Test the logic to automatically configure TOXENV with Travis."""
 
+    def tox_command_raw(self, options):
+        """Return the raw output of finding what tox sees."""
+        proc = subprocess.Popen(
+            ['tox'] + options, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        stdout, stderr = proc.communicate()
+        return proc.returncode, stdout.decode('utf-8'), stderr.decode('utf-8')
+
```

Comment:
I like it. When we get #78 merged, can you make this commit a new PR?

Reply:
Sure.

Reply:
So it looks like you had the same idea in #63, adding a `call_raw` method.